.github/workflows/lint.yaml

```diff
@@ -11,7 +11,10 @@ on:
   pull_request:
     types: [opened, synchronize, reopened]
     branches:
-      - main
+      - '*'
+    paths:
+      - '**.{js,ts,java}'
```

Comment:
let's also do .xml and json? since we might change a lint config/dependency, and we want to know it breaks something (as is for example the case in #422 )

Reply:
And `yml` itself probably?

Reply:
maybe, so uhm, all files then?

Reply:
I think one of our hinters also check license headers in rascal? or is that during mvn build?

Reply:
That's `maven`. But still, I'll remove the file condition here, as it might cause the linter not to be ran unintentionally.